pkgs/tools/misc/nagstamon/default.nix

```
@@ -1,25 +1,57 @@
-{ lib, fetchurl, pythonPackages }:
+{
+  lib,
+  fetchFromGitHub,
+  pythonPackages,
+  qt6,
+}:
 
 pythonPackages.buildPythonApplication rec {
   pname = "nagstamon";
-  version = "3.14.0";
+  version = "3.16.2";
 
-  src = fetchurl {
-    url = "https://github.com/HenriWahl/Nagstamon/archive/refs/tags/v${version}.tar.gz";
-    sha256 = "sha256-9RxQ/rfvoyjSUsY4tmAkBdVQqZYi3X6PBzQYFIeenzA=";
+  src = fetchFromGitHub {
+    owner = "HenriWahl";
+    repo = "Nagstamon";
+    rev = "v${version}";
```

```suggestion
    rev = "refs/tags/v${version}";
```
This prevents branch/tag mismatch.